packages/components/src/lib/controllers/app-controller.js

```diff
@@ -1168,7 +1173,7 @@ export default class AppController {
               we could still offer the option to search the usage examples).
               Similarly, we might prompt the user to create their own analysis...
          */
-        if (data.homonym) { this.showLanguageInfo(data.homonym) }
+        this.showLanguageInfo(data.homonym)
```

Comment:
The presence of this check was preventing the "change language" message to be displayed in the updated lexical query workflow. Situations when `homonym` is undefined can be handled by `shoeLanguageInfo()` without any issues. So there is no reason for this check to be here.